gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,4 @@
-#Sat Jan 11 07:01:08 BRT 2020
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.9-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.3-bin.zip
```

Comment:
Why?
I am pretty sure this is not needed.

Reply:
like im having tons of versions of gradle just because every project use different version and different classifier
btw this is mdk recommended version